docs/custom-images-contract.rst

```diff
@@ -95,10 +95,39 @@ Job arguments, in JSON format.
 .. code-block:: javascript
 
     {"key": "1111112/2/2", "project": 1111112, "version": "version1",
-    "spider": "spider-name", "spider_type": "auto", "tags": [],
-    "priority": 2, "scheduled_by": "user", "started_by": "admin",
+    "spider": "spider-name", "spider_type": "auto", "tags": ["tagA", "tagB"],
+    "priority": 2, "scheduled_by": "user", "started_by": "john",
     "pending_time": 1460374516193, "running_time": 1460374557448, ... }
 
+
+Some useful fields
+__________________
+
+============ ======================================================== =================================
+Field        Description                                              Example
+============ ======================================================== =================================
+key          Job key in format ``PROJECT_ID/SPIDER_ID/JOB_ID``        ``"1111112/2/2"``
+project      Integer project ID                                       ``1111112``
+spider       String spider name                                       ``"spider-name"``
+spider_args  Spider args dictionary                                   ``{"arg1":"val1"}``
+version      String project version used to run the job               ``"version1"``
+units        Amount of units used by the job                          ``1``
+priority     Job priority value                                       ``2``
+tags         List of string tags for the job                          ``["tagA", "tagB"]``
```

Comment:
@jdemaeyer You're right - I was husty with this change, decided that the example is not clear enough w/o the JSON word, thanks for pointing it out. `tags` is a regular list after decoding indeed, as well as other compound structures (fields `job_cmd` is decoded as a list, `spider_args` as a regular dictionary) are decoded correspondingly. Examples are correct, I'm just not sure about proper description and how to represent the examples to make its format obvious enough. If current examples representation is clear then let's just drop `JSON` word from descriptions, do you agree?